index.js

```diff
@@ -157,10 +157,59 @@ function displayFooter() {
 }
 
 function activateSortBtns() {
-  sortBtns.forEach((btn) => btn.addEventListener("click", handleSortBtnClick));
+  sortBtns.forEach((btn) => {
+    btn.addEventListener("click", handleSortBtnClick);
+    btn.sortingState = null;
+  });
 
   function handleSortBtnClick(e) {
-    console.log(e.target.value);
+    let sortedPosts;
+    sortBtns.forEach((btn) => {
+      if (btn.value !== e.target.value) {
+        btn.sortingState = null;
+      }
+    });
+    let [sortBtn, sortedBy] = [e.target, e.target.value];
```

Comment:
I don't think this syntax is necessary..
Moreover, I'd add sortOrder variable.
```suggestion
    const sortBtn = e.target,
              sortBy = sortBtn.value,
              currentSortOrder = sortBtn.sortingOrder;
```

Reply:
will be fixed